pull-requests.md

```diff
@@ -1,63 +1,148 @@
+
+PR approval allows the addition of a general comment that will be displayed in the conversation tab.
+
+Note: If a different reviewer has requested a code change using the “request change” option, you will not be able to approve the PR.  It is effectively blocked until the requester has approved the PR.
+
+# Merging is not the end!
+A PR is not finished when you merge it - check your PR in Production after it's deployed! Although unit & integration tests can add a lot of confidence that your change is good, looking at your change in Production is the ultimate ,and most important place to check your PR is delivering what's wanted.
+
+For webapps you can use [Prout](https://github.com/guardian/prout) to let you know when your change has reached Production, and is ready to be looked at.
+
+A good way to get into the habit of doing post-merge checks is to write one final comment on your PR, detailing what you looked at:
+
+## Writing an informative post-merge PR comment
+A good post-merge comment on a PR can provide evidence that the PR works, or showcase the benefits of doing the PR in the first place. It can answer questions future developers may have, like "Did this stuff _ever_ work?" or "Did this help? Should I make a similar change in my repo?".
+
+Here are some examples of what you could put in a post-merge comment - if your PR was adding:
 * **Metrics or Logging** : Include a sample or screenshot of the newly gathered data, with a link to the ELK or dashboard to make it easy to get to that data in the future ([example](https://github.com/guardian/ophan/pull/4065#issuecomment-802200900))
 * **A new UI feature** : A screenshot or even video of the feature in action in Production. This doesn't need to be too extensive, because you should already have screenshots/video in the main PR description! If the new feature is being A/B tested, you may want to link to the results. ([example](https://github.com/guardian/ophan/pull/3406#issuecomment-522595859))
 * **Performance improvements** : A graph showing how performance has improved post-deployment! ([example](https://github.com/guardian/ophan/pull/4435#issuecomment-1056778719))
 * **A security/bug fix** : any evidence that the bug is no longer there, eg a screenshot, video, or log-search. ([example](https://github.com/guardian/ophan/pull/2896#issuecomment-415489771))
 
+# Additional resources
+A very good summary of good commit messaging can be found [here](https://www.slideshare.net/jaylett/on-committing-well) and [here](https://tekin.co.uk/2019/02/a-talk-about-revision-histories).
+
+It is worth reading [GitHub's blog post "How to write the perfect pull request"](https://github.com/blog/1943-how-to-write-the-perfect-pull-request), and [How to Make Your Code Reviewer Fall in Love with You](https://mtlynch.io/code-review-love/) as these are offer excellent advice on Pull Requests (PRs).
```

Comment:
I think this is a repetition. Do we want to lean into Markdown and implement footnotes?